packages/frames.js/src/middleware/renderResponse.test.tsx

```diff
@@ -471,15 +460,12 @@ describe("renderResponse middleware", () => {
     const result = await render(context, async () => {
       return {
         image: <div>My image</div>,
-        state: { test: true },
+        state: JSON.stringify({ test: true }),
```

Comment:
sorry if i am misunderstanding, but is the user required to serialize state into a string when they return a FrameDefinition in their frames() call?

Reply:
nope, that is handled by `stateMiddleware` automatically